src/structlog/contextvars.py

```diff
@@ -30,6 +32,10 @@
 STRUCTLOG_KEY_PREFIX = "structlog_"
 STRUCTLOG_KEY_PREFIX_LEN = len(STRUCTLOG_KEY_PREFIX)
 
+async_calling_stack: contextvars.ContextVar[Any] = contextvars.ContextVar(
```

Comment:
This variable should definitively be private and `[FrameType]` instead of `Any`, no?

Reply:
Yeah, I forgot to go back to it. Fixed.